### PR TITLE
systemctl: add 'userctl' multicall mode

### PR DESCRIPTION
**Pull request by bluca**
Shortcut for 'systemctl --user'.

Suggested-by: Christian Brauner <brauner@kernel.org>

### CODE REVIEW

File: src/systemctl/systemctl.c [hunk 1]
```diff
@@ -534,6 +534,9 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
+        if (invoked_as(argv, "userctl"))
+                arg_runtime_scope = RUNTIME_SCOPE_USER;
```
Comment (YHNdnzj) [inline src/systemctl/systemctl.c:538]: Hmm, maybe additionally refuse the `--system` option when running as userctl?

File: man/systemctl.xml [hunk 2]
```diff
@@ -42,6 +43,10 @@
     <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
     for an introduction into the basic concepts and functionality this
     tool manages.</para>
+
+    <para><command>userctl</command> may be used to introspect and control the state of the user instance of
+    <literal>systemd</literal>. It is equivalent to <command>systemctl</command> <option>--user</option>.
```
Comment (YHNdnzj) [inline man/systemctl.xml]: We usually include the option in the <command> block too, since they are actually referenced as a whole.

**Review by YHNdnzj — APPROVED** [review]

File: src/systemctl/systemctl.c (hunk 1 above)
Comment (bluca) [reply src/systemctl/systemctl.c:538]: systemctl --user is also accepted, so I prefer to keep the symmetry there

**Comment by bluca** [comment]
> We have userdbctl already, i find userctl confusingly close to that.

When systemctl was added there was already sysctl, which is just as close

**Comment by poettering** [comment]
> When systemctl was added there was already sysctl, which is just as close

not really. also, when we introduced systemctl we weren't close to own of our names at all, as back then sysctl didn't exist within the systemd realm.

Sorry, but I am not a fan of redundancy in general, but would be OK with it in really convincing cases, but I don't think this one qualifies. In particular as "user control" is not what this tool does. "user control" to me would suggest setting user properties, deleting, creating users. Nothing of which systemd does. "systemctl --user" is an interface for "control the service manager of the user" which is quite different from "control the user"...

So I think this is not convincing at all.

**Comment by bluca** [comment]
There are other such 'redundancies' like systemd-umount and systemd-mount --umount but whatever, I'll just use an alias and maintain a private shell completion script for it

**Pull request closed without merging**